recipes/libcurl/all/test_package/test_package.c

```diff
@@ -14,7 +15,12 @@ int main(void)
   for(proto = id->protocols; *proto; proto++) {
     printf("%s ", *proto);
   }
-  printf("\nversion: %s\nssl version: %s\nfeatures: %d\n", id->version, id->ssl_version, id->features);
+  printf("\nversion: %s\nssl version: %s\n", id->version, id->ssl_version);
+
+  printf("features: ");
+  for(feat = id->feature_names; *feat; feat++) {
+    printf("%s ", *feat);
```

Comment:
To be able to check that in fact the features are available instead of a bitmask (which was deprecated usage)